src/components/Login.js

```diff
@@ -16,6 +16,32 @@ class Login extends Component {
     };
   }
 
+  /*
+  function that checks  if access token is in the local storage.
+  If it is, it will route to announcement page and if it is not,
+  it will route to login page.
+  */
+  async componentDidMount() {
+    const response = await fetch("http://localhost:3001/checkToken", {
+      headers: {
+        authorization: localStorage.get("token"),
+        Accept: "application.json"
+      }
+    });
+    if (response.status === 401) {
+      let path = "/login";
+      this.props.history.push(path);
+    }
```

Comment:
why is this part necessary, I don't think we ever send a 401 error and we would just be redirecting to the same route